README.md

```diff
@@ -1,57 +1,75 @@
 # Project Name
 
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
+This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing app service, routing rules with  caching config exposed on a private vnet
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
 - [Terraform](https://www.terraform.io/downloads.html)
 - [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
-- ...
 
 ### Installation
 
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
+``` shell
+git clone https://github.com/Azure-Samples/frontdoor-appservice-vnet-terraform.git
+cd frontdoor-appservice-vnet-terraform
+```
 
 ### Quickstart
-(Add steps to get up and running quickly)
 
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+``` shell
+az login
+az account set -s <subscription_id>
 
+terraform init
+terraform validate
+terraform plan
```

Comment:
plan and apply need to use the terraform.tfvars file `terraform apply -var-file="terraform.tfvars"`

Reply:
Done!